src/components/views/PlayWhites.js

```diff
@@ -9,35 +9,30 @@ import PropTypes from "prop-types";
 for playing white cards on a black Card
  */
 const PlayWhites = () => {
-  // use react-router-dom's hook to access the history
-  const[blackCard, setBlackCard] = useState(null)
-  const[gameId, setGameId] = useState(null)
-  const [cards, setCards] = useState(null)
+    // use react-router-dom's hook to access the history
+    const[blackCard, setBlackCard] = useState(null)
+    const[gameId, setGameId] = useState(null)
+    const [cards, setCards] = useState(null)
     const [count, setCount] = useState(0)
-  const userId = localStorage.getItem("id")
-  const token = localStorage.getItem("token")
-  const cardsPerHand = 8 //constant of how many cards there are in a users hand
+    const userId = localStorage.getItem("id")
+    const token = localStorage.getItem("token")
+    const cardsPerHand = 8 //constant of how many cards there are in a users hand
     // function defines what is happening, when a white card gets selected. also renders the white cards
     const WhiteCard = ({card}) => {
 
         // put the white Card and userId to the game on a Server, reloads useEffects
         const selectCard = async ()  => {
-            if (window.confirm("Press OK to confirm this card")){
-                let cardId = card.id
-                const requestBody = JSON.stringify({gameId});
-                // card gets played
-                try {
-                    await api.post(`users/${userId}/whiteCards/${cardId}`, requestBody);
-                } catch (error) {
-                    console.error("Details:", error);
-                    alert("Invalid Input:\n " + handleError(error));
-                }
-                // next card gets displayed, use for statistic, reloads useEffects
-                setCount(count + 1)
-            }
-            else{
-                console.log("Card was not played!")
+            let cardId = card.id
+            const requestBody = JSON.stringify({gameId});
+            // card gets played
+            try {
+                await api.post(`users/${userId}/whiteCards/${cardId}`, requestBody);
+            } catch (error) {
+                console.error("Details:", error);
+                alert("Invalid Input:\n " + handleError(error));
```

Comment:
Also for the future, just that you know. Since api.post returns a promise, we can handle the response like one. So we can do it like this: ``api.post(..).then(res => {//do something with response}).catch(error => {//do something with error}``